### ISSUE TITLE
Unable to find Images based on Type or Accessibility Accommodations

### ISSUE THREAD

**Issue by clapierre** [benetech/ImageShare#187]
The Frog Resource has 2D tactile graphics as shown here:
![Screen Shot 2020-07-15 at 10 26 25 AM](https://user-images.githubusercontent.com/7736749/87576235-e699a380-c685-11ea-8581-83a20bc0b183.png)

However when I do a search for Type: 2D Tactile this resource doesn't show up as shown here in this screen shot.

![Screen Shot 2020-07-15 at 10 27 08 AM](https://user-images.githubusercontent.com/7736749/87576291-fdd89100-c685-11ea-82c4-71e7b6276ed9.png)

**Comment by clapierre** [comment]
This occurs after - reimporting the same resource.
Both the resource and its associated files change their status to "pending".
I go into the resource and change the reimported pending resource to "published" but its associated files are still pending which is issue #184.
I then go in the resource files and mark all the "pending" files as "published" so now we see each resources associated files, however advance search for "Type" and "Accessibility Accommodation" no longer works.

**Comment by clapierre** [comment]
As mentioned in #184 if both the resource and its associated files are "draft" and then making the "draft" resource "published" results in the correct behavior of both resource and all associated files as being marked published AND the linkage for searching file type/accommodations which this issue addresses.

If you publish separately resource files from its resource this advance search of type and file accommodations does not work.

**Comment by jkva** [comment]
Sorted by fixing #184 and rebuilding the search index.
